.github/workflows/build-test-measure.yml

```
@@ -502,7 +502,10 @@ jobs:
             composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/phpunit:^8.5 --with-dependencies
             echo "Downgrading phpunit/php-token-stream because the latest version requires PHP 7.3"
             composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/php-token-stream:^3.1.3 --with-dependencies
-          elif [[ $WP_VERSION == "latest" || $WP_VERSION == "trunk" || $PHP_VERSION == "7.4" ]]; then
+          elif [[ $PHP_VERSION == "7.3" || $PHP_VERSION == "7.4" || $PHP_VERSION == "8.0" ]]; then
+            echo "Installing PHPUnit 9.3"
+            composer require --dev --ignore-platform-reqs phpunit/phpunit:"=9.3" --with-dependencies
```

Should this rather be:

```suggestion
            composer require --dev --ignore-platform-reqs phpunit/phpunit:"=9.3.*" --with-dependencies
```

As this would account for bug fixes for that branch?

---

Nice catch. It should rather be `composer require --dev --ignore-platform-reqs phpunit/phpunit:"9.3.*" --with-dependencies` as `"=9.3.*"` is an invalid version string.